x-pack/plugins/security_solution/public/threat_intelligence/routes.tsx

```diff
@@ -5,30 +5,49 @@
  * 2.0.
  */
 
-import React from 'react';
+import React, { memo } from 'react';
```

Comment:
this is a result of merging the threat_intelligence.tsx file (was under the pages folder)